src/calculateRentalCost.js

```diff
@@ -1,10 +1,27 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
+  const basePriceOfDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const TermDiscount = 20;
+  const basePrice = days * basePriceOfDay;
+
   // write code here
+  if (days >= longTerm) {
+    return basePrice - longTermDiscount;
+  } else if (days < 3) {
+    return basePrice;
+  } else if (days < longTerm) {
+    return basePrice - TermDiscount;
```

Comment:
The condition `days < longTerm` is redundant here because it is already covered by the `else` block. Consider simplifying the logic by removing this `else if` block and directly applying the short-term discount for days between 3 and 6.